src/resources/clients.ts

```diff
@@ -411,6 +411,27 @@ export class Clients extends Resource<{realm?: string}> {
     queryParamKeys: ['scope', 'userId'],
   });
 
+  public evaluateGenerateIdToken = this.makeRequest<
+  {id: string; scope: string; userId: string},
+  object
```

Comment:
According to the source the return type for this one is `Map<String, Object>`, so we can use a more narrow type here in TypeScript as well.
```suggestion
  Record<string, unknown>
```